services/ucs/src/ucs/main.py

```diff
@@ -46,12 +45,8 @@ async def consume_events(run_forever: bool = True):
 
     async with (
         prepare_event_subscriber(config=config) as event_subscriber,
-        prepare_outbox_subscriber(config=config) as outbox_subscriber,
     ):
-        await asyncio.gather(
-            event_subscriber.run(forever=run_forever),
-            outbox_subscriber.run(forever=run_forever),
-        )
+        await asyncio.gather(event_subscriber.run(forever=run_forever))
```

Comment:
```suggestion
        await event_subscriber.run(forever=run_forever)
```
Gather is no longer needed as this is back to awaiting one subscriber

Reply:
That should have already been removed in [d47b2e9](https://github.com/ghga-de/file-services-backend/pull/97/commits/d47b2e91040ff35535dc0428977071fc34a42a54)